src/pwa/components/Contexts/Context.js

```diff
@@ -123,3 +127,16 @@ export default compose(
   })),
   withTheme,
 )(Context);
+
+const SlotBeforeNavbar = inject(({ stores: { connection } }) => ({
+  item: connection.selectedItem,
```

Comment:
I think the more consistent option is to pass the "observable item" as the `item` prop. This object is used not only to check if it is needed to place a slot in that position but to pass that prop to the slot's children using `fillChildProps` in case they need it.

Reply:
For example, `SmartAd` component uses `item.mstId` to build the `tagId` property [here](https://github.com/frontity/frontity/blob/dev-stable/core/packages/ads/shared/components/Ad/SmartAd/index.js#L56L58).

Reply:
I'm fixing this right now!